moonscript/parse.moon

```diff
@@ -32,297 +31,307 @@ Num = Space * (Num / (v) -> {"number", v})
 } = require "moonscript.parse.util"
 
 
-build_grammar = wrap_env debug_grammar, (root) ->
-  _indent = Stack 0
-  _do_stack = Stack 0
+build_grammar_factory = (litmoon=false) ->
+  EmptyLine=mkEmptyLine litmoon
```

Comment:
what do you mean? you use both camelcase (for grammar) and non-camelcase around there, so i was unsure.